README.md

```diff
@@ -8,7 +8,7 @@ The Ruby [OpenTelemetry](https://opentelemetry.io/) client.
 
 - [Getting Started][getting-started]
 - [Contributing](#contributing)
-- [Instrumentation Libraries](#instrumentation-libraries)
+- [Contrib Repository]](#contrib-repository)
```

Comment:
My only thought here is that it's nice to maintain something referring directly to "instrumentation libraries" both here in this link, and in the section below.

Basically I want to keep advertising loudly in big letters that we have a ton of instrumentation. If someone is quickly scanning the readme, I would want that fact to be very obvious. 😀

Reply:
done!